### PR TITLE
Cocoapods removed + SPM support + SBB24 + new libs

### PR DESCRIPTION
**Pull request by kfa-onewelcome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,19 +6,15 @@ example app.
...
 instructions on configuring access to the OneWelcome Cocoapods repository.
+3. Set SPM registry with a following command:
 
->**Note** Don't forget to update the OneWelcome Cocoapods repository with the following command: `pod repo-art update onegini`. If you don't update the repo it may
-be that the SDK dependency cannot be found. If that is the case be sure to execute the command above.
-
-### Setup the Cocoapods dependencies
-1. Run `pod install` to correctly setup the Cocoapods dependencies
-2. Make sure that you open the project referring to `OneWelcomeExampleApp.xcworkspace` in Xcode or AppCode.
+`swift package-registry set --global https://repo.onewelcome.com/artifactory/api/swift/swift-snapshot-local --netrc`
```
Comment (szymon-onewelcome) [inline README.md:17]: What about the info about valid netrc file structure we spoke (login token)?

Also I would put an info about structure inside `~/.swiftpm/configuration/registries.json`

```
{
  "authentication" : {
    "repo.onewelcome.com" : {
      "loginAPIPath" : "/artifactory/api/swift/swift-snapshot-local",
      "type" : "token"
    }
  },
  "registries" : {
    "[default]" : {
      "supportsAvailability" : false,
      "url" : "https://repo.onewelcome.com/artifactory/api/swift/swift-snapshot-local"
    }
  },
  "version" : 1
}

File: SPM/.gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.DS_Store
+/.build
+/Packages
+xcuserdata/
+DerivedData/
+.swiftpm/configuration/registries.json
+.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
+.netrc
```
Comment (szymon-onewelcome) [inline SPM/.gitignore:8]: I believe lines 6-8 are not needed as those files are stored in `~/` not in sided `SPM/`

File: README.md (hunk 1 above)
Comment (kfa-onewelcome) [reply README.md:17]: this is not you should make manually, (even we tried this approach and did not work)

File: SPM/.gitignore (hunk 2 above)
Comment (kfa-onewelcome) [reply SPM/.gitignore:8]: as I told you offline, this is a separate project (swift package) which was added to the project.
btw. it really does not matter for the sake of the Example app.

Comment (szymon-onewelcome) [reply SPM/.gitignore:8]: Ok... for now :)

**Review by szymon-onewelcome — APPROVED** [review]

**Pull request merged**